common/djangoapps/track/tests/test_shim.py

```diff
@@ -62,11 +63,8 @@ def test_event_field_mapping(self):
             'page': None,
             'session': sentinel.session,
         }
-        self.assertEqualUnicode(expected_event, emitted_event)
+        assert_event_matches(expected_event, emitted_event)
```

Comment:
Hmm.  Maybe it is a good idea for the shim to be tested strictly, now that I think of it.  Doesn't the shim deal with a lot of top-level attributes, not just the payload?